internal/kafka/command_broker_list.go

```diff
@@ -48,7 +42,7 @@ func (c *brokerCommand) list(cmd *cobra.Command, _ []string) error {
 
 	list := output.NewList(cmd)
 	for _, data := range brokersGetResp.Data {
-		broker := &brokerOut{
+		broker := &broker.BrokerOut{
```

Comment:
```suggestion
		broker := &broker.Out{
```

Reply:
there're 2 types or `out` structs, config or broker

Reply:
The package is called broker, so we can have: `broker.Out` and `broker.ConfigurationOut`

Reply:
(`broker.BrokerOut` is a bit redundant, IMO)